draft-ietf-taps-impl.md

```diff
@@ -639,20 +639,20 @@ its `Ready` event. This allows the implementation to communicate control data to
 Remote Endpoint that can be used to parse Messages.
```

Comment:
```suggestion
Remote Endpoint that can be used to parse Messages.

Once the framer implementation has completed its setup or handshake, it can indicate to
the application that it is ready to handling data with this call.
```